tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs

```diff
@@ -1173,17 +1279,16 @@ public async Task CanRun_NetCore30App_PublishedOnMacMachine_ButRunOnNetCore30Run
                 new DockerCli());
         }
 
-        [Theory]
-        [InlineData("NetCoreApp30.WebApp")]
```

Comment:
I removed support for making this scenario work as users could always do `./NetCoreApp30.WebApp` and supporting this feature was making the script more complicated

Reply:
we solve this for the other languages by using `common.ExtendPathForCommand` from our common library.
It would be great if the .NET startup gen followed the pattern of the other languages and had its code in Go, so we could benefit from those things we already have in Go

Reply:
As discussed offline, I will work on making majority of the dotnetcore startupscript generation in Golang code and will use the functionality you mention at that time.